### DISCUSSION TITLE
Buenos Dias Explorers

### DISCUSSION THREAD

**Discussion by Tongas5625** [Launch-X-Latam/MisionFrontEnd#448] [Team Posts]
tengo duda con el tema de las clases, que dias son y por donde puedo acceder a las mismas?

**Comment by DanielSilva26** [comment]
Las preguntas se hacen en la comunidad de GitHub como los mission commander pidieron por favor, nos están llenando de SPAM nuestros correos hablando por aquí.

Enviado desde Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: Gastón ***@***.***>
Enviado: Wednesday, October 12, 2022 9:10:17 AM
CC: Subscribed ***@***.***>
Asunto: [Launch-X-Latam/mision-frontend] Buenos Dias Explorers (#17)


tengo duda con el tema de las clases, que dias son y por donde puedo acceder a las mismas?
